docs/google/engine.md

```diff
@@ -139,6 +139,16 @@ number of repetitions.
 Batching circuits together that do not follow these restrictions may not
 cause an error, but your performance will not be significantly improved.
 
+Results can be retrieved in two different forms.  If the resulting
```

Comment:
"the resulting" looks like it should be removed - there's no action described that anything is resulting from.

Reply:
Done.